auth-web/src/store/modules/business.ts

```diff
@@ -1,5 +1,5 @@
 import { Action, Module, Mutation, VuexModule } from 'vuex-module-decorators'
-import { Business, FolioNumberload, LoginPayload, NamedBusinessRequest, NumberedBusinessRequest } from '@/models/business'
+import { Business, BusinessRequest, FolioNumberload, LoginPayload } from '@/models/business'
```

Comment:
Funny, linting alphabetized these imports :)

Reply:
hehe, yes the linters are particular about the imports!